capstone/src/main/java/com/google/univiz/SuggestionDataApi.java

```diff
@@ -0,0 +1,15 @@
+package com.google.univiz;
+
+import java.util.List;
+
+/**
+ * Communicates directly with the REST API to get a list of college suggestions related to a client
```

Comment:
Avoid implementation details in the interface documentation. Today we'll use the REST API but tomorrow it could be a database or something else entirely. If this class stays agnostic to the implementation then the documentation won't need updating for situations like that...

Reply:
I updated the documentation. However, on a related note-should I still be communicating directly with the REST API in the implementation of this interface  (in private functions)?

Reply:
> should I still be communicating directly with the REST API in the implementation of this interface (in private functions)?

Oh yes! And that implementation will be in the `scorecard` package where you can feel free to discuss the specifics of the implementation in JavaDoc :)